File: examples/modulith-clinical-tool-jpa/src/main/java/io/zenwave360/example/clinicaltool/modules/termsandconditions/core/inbound/dtos/AcceptedTermsAndConditionsInput.java
package io.zenwave360.example.clinicaltool.modules.termsandconditions.core.inbound.dtos;

import io.zenwave360.example.clinicaltool.modules.termsandconditions.core.domain.*;
import java.io.Serializable;
import java.math.*;
import java.time.*;
import java.util.*;
import jakarta.validation.constraints.*;

/**
 * AcceptedTermsAndConditionsInput.
 */
@lombok.Getter @lombok.Setter
public  class AcceptedTermsAndConditionsInput  implements Serializable {


    @NotNull
    private Long termsAndConditionsId ;




}

File: examples/modulith-klinical-tool-jpa/src/main/kotlin/io/zenwave360/example/clinicaltool/modules/termsandconditions/dtos/AcceptedTermsAndConditionsInput.kt
package io.zenwave360.example.clinicaltool.modules.termsandconditions.dtos

import io.zenwave360.example.clinicaltool.modules.termsandconditions.domain.*
import jakarta.validation.constraints.*
import java.io.Serializable
import java.math.*
import java.time.*

/** AcceptedTermsAndConditionsInput. */
data class AcceptedTermsAndConditionsInput(
    @NotNull val userId: Long? = null,
    @NotNull val termsAndConditionsId: Long? = null,
) : Serializable {}
